topics/browser-extension/index.md

```diff
@@ -0,0 +1,7 @@
+---
+display_name: Browser extension
+short_description: A browser extension is a small software module for customizing a web browser.
+topic: browser-extension
+related: javascript, typescript
+---
+A browser extension typically allow a variety of extensions, including user interface modifications, cookie management, ad blocking, and the custom scripting and styling of web pages.
```

Comment:
I do think that you should formulate it more like this:
`A browser does typically allow a ...`

Reply:
No more $, #, no more stacking. Replace with your own. One that is unique to you and your device.. mine is similar to what follows but I also programmed each 'bot' to recognize my voice as the url, inside the url. You're the host, domain is the formula for your Unique.Realistic.Lobot..
~star2pltesq/%\pltesq2.2_[phrack{8.8}€]